mlir/lib/Conversion/GPUToLLVMSPV/GPUToLLVMSPV.cpp

```diff
@@ -53,7 +52,19 @@ static LLVM::LLVMFuncOp lookupOrCreateSPIRVFn(Operation *symbolTable,
         symbolTable->getLoc(), name,
         LLVM::LLVMFunctionType::get(resultType, paramTypes));
     func.setCConv(LLVM::cconv::CConv::SPIR_FUNC);
+    func.setNoUnwind(true);
+
+    if (!isMemNone) {
```

Comment:
```suggestion
    if (isMemNone) {
```

Reply:
And change `true<->false` in calling sites. You are doing quite the opposite of what the name suggests.

Reply:
Ooops! Thanks for spotting that.